diff -p -up libdc1394-2.1.2/config.h.in~ libdc1394-2.1.2/config.h.in
--- libdc1394-2.1.2/config.h.in~	2009-06-11 06:07:19.000000000 +0200
+++ libdc1394-2.1.2/config.h.in	2009-09-22 17:47:07.000000000 +0200
@@ -63,6 +63,10 @@
 /* Defined if compiled for Windows */
 #undef HAVE_WINDOWS
 
+/* Define to the sub-directory in which libtool stores uninstalled libraries.
+   */
+#undef LT_OBJDIR
+
 /* Define to 1 if your C compiler doesn't accept -c and -o together. */
 #undef NO_MINUS_C_MINUS_O
 
@@ -81,6 +85,9 @@
 /* Define to the one symbol short name of this package. */
 #undef PACKAGE_TARNAME
 
+/* Define to the home page for this package. */
+#undef PACKAGE_URL
+
 /* Define to the version of this package. */
 #undef PACKAGE_VERSION
 
@@ -97,13 +104,14 @@
    nothing if this is not supported.  Do not define if restrict is
    supported directly.  */
 #undef restrict
-/* Work around a bug in Sun C++: it does not support _Restrict, even
-   though the corresponding Sun C compiler does, which causes
-   "#define restrict _Restrict" in the previous line.  Perhaps some future
-   version of Sun C++ will work with _Restrict; if so, it'll probably
-   define __RESTRICT, just as Sun C does.  */
+/* Work around a bug in Sun C++: it does not support _Restrict or
+   __restrict__, even though the corresponding Sun C compiler ends up with
+   "#define restrict _Restrict" or "#define restrict __restrict__" in the
+   previous line.  Perhaps some future version of Sun C++ will work with
+   restrict; if so, hopefully it defines __RESTRICT like Sun C does.  */
 #if defined __SUNPRO_CC && !defined __RESTRICT
 # define _Restrict
+# define __restrict__
 #endif
 
 /* Define to `unsigned int' if <sys/types.h> does not define. */
--- libdc1394-2.1.2/dc1394/Makefile.am.link	2009-06-11 06:05:47.000000000 +0200
+++ libdc1394-2.1.2/dc1394/Makefile.am	2011-02-05 08:22:43.000000000 +0100
@@ -5,7 +5,7 @@
 AM_CFLAGS = $(platform_CFLAGS) -I$(top_srcdir)
 
 libdc1394_la_LDFLAGS = $(platform_LDFLAGS) \
-	-version-info @lt_current@:@lt_revision@:@lt_age@
+	-version-info @lt_current@:@lt_revision@:@lt_age@ -pthread
 
 libdc1394_la_SOURCES =  \
 	control.c 	\
@@ -51,7 +51,7 @@
 	$(MACOSX_LIBADD) \
 	$(MSW_LIBADD) \
 	$(USB_LIBADD) \
-	vendor/libdc1394-vendor.la
+	vendor/libdc1394-vendor.la -lpthread
 
 # headers to be installed
 pkginclude_HEADERS =  	\
